stablehlo/dialect/StablehloOps.td

```diff
@@ -2468,9 +2468,9 @@ def StableHLO_DynamicReshapeOp: StableHLO_ShapedInterfaceOp<"dynamic_reshape", [
   let assemblyFormat = "operands attr-dict `:` functional-type(operands, results)";
 }
 
-def StableHLO_ScatterOp: StableHLO_Op<"scatter",
-      [SameVariadicOperandSize, RecursiveMemoryEffects,
-      DeclareOpInterfaceMethods<InferTypeOpInterface>]> {
+def StableHLO_ScatterOp: StableHLO_Op<"scatter", [RecursiveMemoryEffects,
+      SameVariadicOperandSize /*scatter_c1, scatter_c3, scatter_c5*/,
```

Comment:
c1 and c3 are about input and updates have the same shape. I think only c5 is applicable here.